src/router/index.js

```diff
@@ -33,7 +33,7 @@ const routes = [
         component: Cart,
       },
       {
-        path: "products/:name",
+        path: ":name",
```

Comment:
I am not sure how we are going to connect the detail view with the products' view. Maybe we can also pass the id of the product in the path, and the detail view could fetch the product data from the backend with this id.
However, I don't know if you have another idea in mind so let me know what you think.